### PR TITLE
59 map plugin

### PR DESCRIPTION
**Pull request by ianwood103**
### Testing:
- cd into eos-map
- run npm install
- run npm start

*Data will need to be in Postgres db for it to display any data*

### CODE REVIEW

**Review by IonLi09 — COMMENTED** [review]
Not sure if this is a problem on my side, but the map isn't showing up on my end. There are a few ways to get it to display if it's not just me

**Comment by IonLi09** [comment]
Also, update the EosLib requirement to 4.1.1

File: postgresDB/postgresDB/settings.py [hunk 1]
```diff
@@ -86,7 +88,7 @@
         },
         'NAME': 'eos_db',
         'USER': 'postgres',
-        'PASSWORD': '',
+        'PASSWORD': 'password',
```
Comment (IonLi09) [inline postgresDB/postgresDB/settings.py:91]: security breach!!

**Review by IonLi09 — APPROVED** [review]
lgtm

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/VIP-LES/EosLib@v3.0.0#egg=EosLib
+git+https://github.com/VIP-LES/EosLib@v4.1.1#egg=EosLib
```
Comment (IonLi09) [inline requirements.txt:1]: mb i didn't see this in the slack. Aryan bumped version to 4.1.2

**Review by IonLi09 — APPROVED** [review]
lgtm, test it with the full pipeline before merging

**Review by VanL15 — APPROVED** [review]
lgtm

**Pull request merged**